packages/renderer/src/lib/pod/KubernetesTerminal.spec.ts

```diff
@@ -19,7 +19,10 @@
 import '@testing-library/jest-dom/vitest';
 
 import { render, waitFor } from '@testing-library/svelte';
+/* eslint-disable import/no-duplicates */
+import { tick } from 'svelte';
 import { get } from 'svelte/store';
+/* eslint-enable import/no-duplicates */
```

Comment:
No duplicates ?

Reply:
the no-duplicates rule is for svelte imports

by default the plug-in by seeing 
```
import { tick } from 'svelte';
import { get } from 'svelte/store';
```

will migrate to
```
import {get, tick} from 'svelte'
```

due so some exposures and it'll fail this is why all imports when importing both from the core and the store need to be in a no-duplicates block 

it's not related to svelte5, we already do this and there is a bug referenced somewhere for svelte/the plugin

Reply:
the original issue https://github.com/import-js/eslint-plugin-import/issues/1479

Reply:
Thanks for the details